pkgs/applications/audio/nova-filters/default.nix

```diff
@@ -16,6 +16,7 @@ stdenv.mkDerivation rec {
     sed -i -e '4d' SConstruct
     sed -i "s@mfpmath=sse@mfpmath=sse -I ${boost.dev}/include@g" SConstruct
     sed -i "s@ladspa.h@${ladspaH}/include/ladspa.h@g" filters.cpp
+    sed -i "s@LADSPA_HINT_SAMPLE_RATE, 0, 0.5@LADSPA_HINT_SAMPLE_RATE, 0.0001, 0.5@g" filters.cpp
```

Comment:
Sorry for omitting that.
In the original code the user set frequencies have a range of 0 to samplerate/2.
At the same time the scaling is set to log, cause it's a frequency.
Since log 0 doesn't make sense here, I set the lower frequency to  0.0001*samplerate.
Not sure how elaborate I should be in my comment.